package.json

```diff
@@ -3,12 +3,19 @@
   "description": "Realtime client library for Ably.io, the realtime messaging service",
   "version": "0.9.0-beta.0",
   "main": "./nodejs/index.js",
+  "react-native": {
+    "./nodejs/index.js": "browser/static/ably-reactnative.js"
+  },
+  "browser": {
+    "./nodejs/index.js": "browser/static/ably-commonjs.js"
+  },
   "dependencies": {
     "async": "git://github.com/ably-forks/async#requirejs",
     "hexy": "~0.2",
     "msgpack-js": "git://github.com/paddybyers/msgpack-js.git",
     "request": "^2.74.0",
-    "ws": "^1.1.1"
+    "ws": "^1.1.1",
+    "react-native-randombytes": "git+https://github.com/ably-forks/react-native-randombytes.git"
```

Comment:
> This is quite unfortunate. I think it's OK for this release, but we should add a task to create a new repo for ably-reactive-native that simply depends on ably-js and adds the necessary dependency, WDYT?

Maybe I'm not following but I still don't understand why it's not a dev dependency.

Since this only a spurious dependency in node deployments I don't think the impact of including it is that bad, even if it's a bit crap.